### PR TITLE
WIP: Doc fixes

### PR DESCRIPTION
**Pull request by ksbeattie**
Been walking thru trying to fix various little reference link problems in the docs....

The results of this (on-going) work should be viewable at:
https://lbl-ksbeattiefoqus.readthedocs-hosted.com/en/doc_fixes/

### CODE REVIEW

**Comment by ksbeattie** [comment]
@sotorrio1, @fsoepyan, @anujad95, @eslickj: there are several references to `ap:psuadefiles` and I'm not sure exactly where that reference should point.  Can someone suggest where it should point?

Here they are:
[./chapt_uq/tutorial/sim.rst](../blob/master/docs/source/chapt_uq/tutorial/sim.rst) lines: [65](../blame/master/docs/source/chapt_uq/tutorial/sim.rst#L65), [137](../blame/master/docs/source/chapt_uq/tutorial/sim.rst#L137) and [156](../blame/master/docs/source/chapt_uq/tutorial/sim.rst#L156)

 [./chapt_uq/reference.rst](../blob/master/docs/source/chapt_uq/reference.rst) lines: [90](../blame/master/docs/source/chapt_uq/reference.rst#L90), [422](../blame/master/docs/source/chapt_uq/reference.rst#L422) and [529](../blame/master/docs/source/chapt_uq/reference.rst#L529)

**Comment by sotorrio1** [comment]
I'm not sure about those references to `ap:psuadefiles` but I can try to take a look and find out what's going. The same with the rest of references in the UQ chapter.

**Comment by sotorrio1** [comment]
@ksbeattie I've been going through an old version of the documentation (when we used to use LaTeX and created PDF from it). All the references to `ap:psuadefiles` are referencing a section within the UQ chapter that it was right after the tutorials called "File Formats". In that section it explains everything about the different PSUADE file formats. What I don't understand is why that section didn't make to the readthedocs version...

**Comment by ksbeattie** [comment]
@sotorrio1 will look for something to reference directly in the PSUADE documentation
@ksbeattie will look if it might be easy to ressurect the old LaTeX docs for inclusion in our docs.

**Comment by sotorrio1** [comment]
There is nothing similar to the original section 4.3 File Formats in the PSUADE documentation. There are some references to the PSUADE command `showformat` in the [PsuadeRefManual.pdf](https://github.com/LLNL/psuade/blob/master/Doc/Manual/PsuadeRefManual.pdf). When you run that command within PSUADE it gives you an idea of the format you need.

File: docs/source/chapt_opt/reference/optimization.rst [hunk 1]
```diff
@@ -271,7 +267,7 @@ returned as a list with only one element. The last two return values are
 debugging information for constraints. In this case, the “zeros” are
 just place holders and have no real utility.
 
-.. code-block::
+.. code-block:: python
```
Comment (ksbeattie) [inline docs/source/chapt_opt/reference/optimization.rst]: This is the fix for #503

**Comment by ksbeattie** [comment]
I'm going to merge this now and continue this in a future PR.  Capturing the effort to clean up broken references in the docs in issue #605

**Pull request merged**